Hakawai/Mentions/HKWMentionsPlugin.m

```diff
@@ -1426,7 +1469,89 @@ -(void) textViewDidProgrammaticallyUpdate:(UITextView *)textView {
     [self resetAuxiliaryState];
 }
 
+- (void)textView:(UITextView *)textView shouldChangeTextInRange:(NSRange)range replacementText:(NSString *)text isInsertion:(BOOL)isInsertion {
```

Comment:
this is almost equivalent to the shouldChangeTextInRange override, with:
a programatic previousTextLength calculation;
passed-through deleted text, because NSTextStorageDelegate is called after textview is updated to already have removed the character(s);
isInsertion being the way that we calculate whether something is an insertion or not